### PR TITLE
[WIP]Select language for input type literals, defaults to English

### LINKED ISSUE #263: when user is inputing a literal value, they should be presented with the option of selecting a language
a literal field should have some way to indicate language.  This could be a pulldown or a QA lookup to languages.  It should appear in the RDF as _@en_ after the literal, such as 

```
<http://example.org/object/666> rdfs:label "notated music"@en.
```

### PR DESCRIPTION
**Pull request by SaravShah**
closes #263 

Implementing the Language Button:

Default is set to English

After you enter text into the input literal field:

screen shot 2019-01-10 at 3 06 22 pm

Next click on the English button:
screen shot 2019-01-10 at 3 06 27 pm

Then start typing language in: 
screen shot 2019-01-10 at 3 06 35 pm

Click submit and see language as been updated:
screen shot 2019-01-10 at 3 08 04 pm

### CODE REVIEW

File: src/components/editor/InputLiteral.jsx [hunk 1]
```diff
@@ -98,27 +116,78 @@ export class InputLiteral extends Component {
...
+      return "English"
+    } else{
+      return newLang
+    }
+  }
+
+  // Bug
+  // When adding the same value twice to the input field, the languages concat and then we have a problem.
+  // English is the default value, but is not set in the redux.lang.store. Needs to be set manually in the
+  // generation of RDF.
```
Comment (ndushay) [inline src/components/editor/InputLiteral.jsx]: are there tickets for the two issues above?  and for the one below?

File: src/store.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { createStore } from 'redux'
 import reducer from './reducers/index'
 
-const store = createStore(reducer,
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
+const store = createStore(reducer)
+  // add the following to createStore if want to use Redux plug in
```
Comment (ndushay) [inline src/store.js]: add "(e.g. for debugging)" ?

File: src/components/editor/InputLiteral.jsx (hunk 1 above)
Comment (SaravShah) [reply src/components/editor/InputLiteral.jsx]: creating them now

Comment (ndushay) [reply src/components/editor/InputLiteral.jsx]: put tix # into comments?  Or is that too picky?

**Comment by jgreben** [comment]
Rebased and created new PR: https://github.com/LD4P/sinopia_editor/pull/328

**Pull request closed without merging**